README.md

```diff
@@ -1,7 +1,6 @@
 # PostgreSQL Virtual Schema
 
-An implementation of the [PostgreSQL](https://www.postgresql.org/) dialect for the [Exasol Virtual Schema JDBC][virtual-schema-common-jdbc] project.
-Use this dialect if you want to create a Virtual Schema based on a PostgreSQL database.
+An implementation of the [PostgreSQL](https://www.postgresql.org/) dialect for the [Exasol Virtual Schema JDBC][virtual-schema-common-jdbc] project. Use this dialect if you want to create a Virtual Schema based on a PostgreSQL database.
```

Comment:
Let's unify this paragraph with other VS, you can take a template from here, for example 
https://github.com/exasol/mysql-virtual-schema